translations/de_DE.json

```diff
@@ -0,0 +1,612 @@
+{
+  "ABOUT": "Über",
+  "ACTIVE": "Aktiv",
+  "ADD": "Hinzufügen",
+  "ADDED": "hinzugefügt",
+  "ADD_FOLDER": "Ordner hinzufügen",
+  "ADD_LANGUAGE": "Sprache hinzufügen",
+  "ADD_LANGUAGE_SUBTITLE": "Hilf Namida in deine Sprache zu übersetzen",
+  "ADD_LRC_FILE": "LRC-Datei hinzufügen",
+  "ADD_MORE_FROM_THIS_ALBUM": "Mehr aus diesem Albums zur Warteschlange hinzufügen",
+  "ADD_MORE_FROM_THIS_ARTIST": "Mehr von diesem Interpreten zur Warteschlange hinzufügen",
+  "ADD_MORE_FROM_THIS_FOLDER": "Mehr aus diesem Ordner zur Warteschlange hinzufügen",
+  "ADD_MORE_FROM_TO_QUEUE": "Add more from _MEDIA_ to queue",
+  "ADD_TO_PLAYLIST": "Zur Playlist hinzufügen",
+  "ADVANCED": "Erweitert",
+  "ADVANCED_SETTINGS": "Erweitert",
+  "ADVANCED_SETTINGS_SUBTITLE": "Erweiterte Einstellungen, nicht anfassen!",
+  "AGO": "ago <- fix this",
```

Comment:
@MSOB7YY pls fix :)

Reply:
yeah this is not used anymore after relying on [Jiffy](https://pub.dev/packages/jiffy) library for times.
this will get removed along with other un-used keys:
- [x] ~AGO~
- [ ] ALBUM_ARTISTS (need soon)
- [ ] CORRUPTED_FILE (gonna need)
- [ ] GO_TO_CHANNEL (needed)
- [x] ~GRANT_ACCESS~ (now `GRANT_STORAGE_PERMISSION` is used)
- [x] ~MIN_FILE_DURATION_SUBTITLE~
- [x] ~MIN_FILE_SIZE_SUBTITLE~
- [x] ~PERMISSION_UPDATE~
- [ ] SUBSCRIBED (needed)
- [ ] SUSSY_BAKA
- [ ] THEME_MODE_LIGHT  (might be needed)
- [ ] THEME_MODE_SYSTEM  (might be needed)
- [ ] VIDEO_PLAYBACK_SOURCE_LOCAL_EXAMPLE_SUBTITLE (still not sure)
- [x] ~VIDEO_PLAYBACK_SOURCE_LOCAL_EXAMPLE~
- [x] ~WAVEFORMS~ (old memories)

#### thanks for pointing this out